backend/models/Node.js

```diff
@@ -65,7 +65,7 @@ module.exports = (sequelize, DataTypes) => {
       tableName: "nodes",
       underscored: true,
       timestamps: false,
-      indexes: [{ fields: ["last_seen"] }, { fields: ["is_validator"] }],
+      indexes: [{ fields: ["node_id"] }, { fields: ["is_validator"] }],
```

Comment:
1. `node_id` is a primary field, so you don't need to create another index on it.
2. `last_seen` is used to determine the online status, so we can keep it indexed.